cli-commands/forgae-test/test.js

```diff
@@ -35,7 +34,7 @@ const run = async (path) => {
   }
 
   testDirectory = `${process.cwd()}/${path}`;
-  const files = await utils.getFiles(`${process.cwd()}/${path}/`, `.*\.(js|es|es6|jsx|sol)$`);
+  const files = await utils.getFiles(`${process.cwd()}/${path}/`, `.*\\.(js|es|es6|jsx|sol)$`);
```

Comment:
What is the change here? Also what is sol doing here?

Reply:
when you write a regex in js '\' should be escaped too. if you use '\.', the back slash do not escape dot(.) and match everything

Reply:
.sol was there ...